### PR TITLE
FW-5138 Add order by search parameter

### PR DESCRIPTION
**Pull request by NolanVH**
### Description of Changes
This PR adds the ability to order the search results by the following:
- `created`: orders results by the created timestamp
- `modified`: orders results by the last modified timestamp
- `title`: orders results by title based on the alphabet custom order

Results are sorted in ascending order by default and `_desc` can optional get added to the parameter which will sort results in descending order (eg: `sort=created_desc`).

The created and last_modified values have been added to the search index to support these parameters. Additionally, the `created` and `last_modified` values have been added to the serializers used by the search view.

### Checklist
- ~README / documentation has been updated if needed~
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- ~Admin Panel has been updated if models have been added or modified~
- ~Migrations have been updated and committed if applicable~
- [x] Team Postman workspace has been updated if applicable

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

**Review by sarahfirstvoices — CHANGES_REQUESTED** [review]

**Review by bronson-c — APPROVED** [review]
LGTM! 👍

File: firstvoices/backend/search/utils/query_builder_utils.py [hunk 1]
```diff
@@ -278,3 +278,18 @@ def get_valid_visibility(input_visibility_str):
...
+    descending = len(input_string) > 1 and input_string[1] == "desc"
+
+    if len(input_string) > 0 and (
+        input_string[0] == "created"
+        or input_string[0] == "modified"
+        or input_string[0] == "title"
+    ):
+        return input_string[0], descending
+    else:  # if invalid string is passed
+        return None, descending
```
Comment (sarahfirstvoices) [inline firstvoices/backend/search/utils/query_builder_utils.py:295]: This is fine, but I don't think we really need to return the descending value here

Comment (sarahfirstvoices) [reply firstvoices/backend/search/utils/query_builder_utils.py:295]: I take it back about this being fine-- I think we should return None, None here to eliminate the possibility of a bug that would sort by reverse search score

File: firstvoices/backend/views/search/base_search_views.py [hunk 2]
```diff
@@ -379,10 +425,44 @@ def list(self, request, **kwargs):
             from_=pagination_params["start"], size=pagination_params["page_size"]
         )
 
-        # Sort by score, then by custom sort order
-        search_query = search_query.sort(
-            "_score", {"custom_order": {"unmapped_type": "keyword"}}, "title.raw"
+        sort_direction = (
+            "desc"
+            if search_params["descending"] and search_params["sort"] is not None
```
Comment (sarahfirstvoices) [inline firstvoices/backend/views/search/base_search_views.py:430]: If you always return None for descending when the sort param is missing or invalid, you can skip this type of check on the sort param value

**Review by sarahfirstvoices — APPROVED** [review]
Thanks for making all these changes! I'm approving it, but please do update that one return value

**Pull request merged**